### PR TITLE
Use -1 instead of 0 to disable the idle timeout mechanism

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
- [x] ⚠️ Merge after releasing https://github.com/iterative/cml/pull/700

### CODE REVIEW

File: content/docs/self-hosted-runners.md [hunk 1]
```diff
@@ -201,8 +201,7 @@ machine or on-premise GPU cluster.
 cml-runner \
   --repo="$repository_url" \
   --token="$personal_access_token" \
-  --labels="local,runner" \
-  --idle-timeout=0
+  --labels="local,runner"
```
Comment (casperdcl) [inline content/docs/self-hosted-runners.md:204]: this change would be OK if we address https://github.com/iterative/cml/pull/700#issuecomment-902926901

File: content/docs/self-hosted-runners.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ The function `cml-runner` accepts the following arguments:
...
 | `--version`                       | N/A                                                        | Show version number and exit (ignoring all other options)                                                                                             |
 | `--labels <list>`                 | `cml`                                                      | One or more user-defined labels for this runner (delimited with commas)                                                                               |
-| `--idle-timeout <seconds>`        | `300`                                                      | Time in seconds for the runner to be waiting for jobs before shutting down. Setting it to `0` disables automatic shutdown                             |
+| `--idle-timeout <seconds>`        | `300`                                                      | Seconds to wait for jobs before shutting down. Set to -1 to disable timeout                                                                           |
```
Comment (casperdcl) [inline content/docs/self-hosted-runners.md:90]:
```suggestion
| `--idle-timeout <seconds>`        | `300`                                                      | Seconds to wait for jobs before shutting down. Set to `-1` to disable timeout                                                                         |
```

File: content/docs/self-hosted-runners.md (hunk 1 above)
Comment (casperdcl) [reply content/docs/self-hosted-runners.md:204]: this probably should be reverted...

Comment (0x2b3bfa0) [reply content/docs/self-hosted-runners.md:204]: 👍🏼

Comment (0x2b3bfa0) [reply content/docs/self-hosted-runners.md:204]: > _leave --idle-timeout=-1_

:this:

Comment (casperdcl) [inline content/docs/self-hosted-runners.md:204]:
```suggestion
  --labels="local,runner" \
  --idle-timeout=-1
```

**Review by casperdcl — APPROVED** [review]

**Pull request merged**